tools.sh

```diff
@@ -32,7 +32,7 @@ esac
 # Download yq/jq for manipulating in place yaml configs
 if ! "${YQ}" -V; then
     if [[ ! -e yq ]]; then
-        curl -L https://github.com/mikefarah/yq/releases/download/3.3.0/yq_linux_${yq_ARCH} -o yq
+        curl -L https://github.com/mikefarah/yq/releases/download/v4.5.1/yq_linux_${yq_ARCH} -o yq
```

Comment:
What about users who already have yq installed? Is the new syntax you are using backwards compatible with 3.x? Or does `yq` have to be replaced before snc.sh can run?

Reply:
@cfergeau I will try to change it something we are doing for `openshift-install` command, always download if not specified.